src/planner/match/MatchClausePlanner.cpp

```diff
@@ -297,6 +304,35 @@ Status MatchClausePlanner::appendFetchVertexPlan(const Expression* nodeFilter,
     return Status::OK();
 }
 
+StatusOr<std::vector<storage::cpp2::VertexProp>>
+MatchClausePlanner::flattenTags(QueryContext *qctx, const SpaceInfo& space) {
+    // Get all tags in the space
+    const auto allTagsResult = qctx->schemaMng()->getAllVerTagSchema(space.id);
```

Comment:
the `getAllVerTagSchema` will get all version tags, and the result will include different version tags, I think you need to get the lastest tag, not all version tags.